webapp/WEB-INF/conf/core_context.xml

```diff
@@ -50,7 +53,10 @@
     <bean id="siteMessageHandler" class="fr.paris.lutece.portal.service.message.SiteMessageHandler"/>
 
     <!--                      Search Engine                                   -->
-    <bean id="searchEngine" class="fr.paris.lutece.portal.service.search.LuceneSearchEngine" />
+    <bean id="searchEngine" class="fr.paris.lutece.portal.service.search.LuceneSearchEngine">
+        <aop:scoped-proxy proxy-target-class="false" />
```

Comment:
I can't find any docs on using aop-proxy this way. Other people have done it though: http://stackoverflow.com/questions/3180963/spring-session-scoped-beans-controllers-and-references-to-services-in-terms-o
Is it totally undocumented ? If so, maybe we should just keep the local variable and the call to getBean on each method call ?